tests/todomanager-tests.js

```diff
@@ -0,0 +1,11 @@
+const assert = require('assert');
+const eoslime = require('eoslime').init('local');
+
+const TODO_MANAGER_WASM_PATH = '../contracts/todomanager.wasm';
+const TODO_MANAGER_ABI_PATH = '../contracts/todomanager.abi';
+
+describe('TODO Manager', function () {
```

Comment:
eoslime test will change `describe('TODO Manager', function () {` to `describe('TODO Manager', function (eoslime) {`

Reply:
fixed